scripts/common_cmdline.sh

```diff
@@ -74,17 +78,37 @@ function compileFull()
     if [[ \
         ("$exit_code" -ne "$expected_exit_code" || \
             ( $expect_output -eq 0 && -n "$errors" ) || \
-            ( $expect_output -ne 0 && $expected_exit_code -eq 0 && $expect_output -ne 2 && -z "$errors" ))
+            ( $expect_output -ne 0 && $expected_exit_code -eq 0 && \
+              $expect_output -ne 2 && -z "$errors" )
+        )
     ]]
     then
-        printError "Unexpected compilation result:"
-        printError "Expected failure: $expected_exit_code - Expected warning / error output: $expect_output"
-        printError "Was failure: $exit_code"
-        echo "$errors"
+        printError "Test failure:"
+        printError "Exit code: $exit_code"
+        printError "Expected:  $expected_exit_code"
```

Comment:
```suggestion
        printError "Expected output:  $expected_exit_code"
```

Reply:
I specifically didn't write output here because it's the exit code not the output.. it previously was like that and I found it extremely confusing...

Reply:
Right. Not sure why I actually suggested "expected output" here. Looking at it now, it makes no sense. I might have meant "expected exit code". Anyway, I see you're already using "expected exit code" there so it's fine. Marking as resolved.